ledger_sync-xero.gemspec

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+lib = File.expand_path("../lib", __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require "ledger_sync/ledgers/xero/version"
+
+Gem::Specification.new do |spec|
+  spec.name = "ledger_sync-xero"
+  spec.version = LedgerSync::Ledgers::Xero::VERSION
+  spec.authors = ["Jozef Vaclavik"]
+  spec.email = ["jozef@hey.com"]
+
+  spec.summary = 'Sync common objects to accounting software.'
+  spec.description = 'LedgerSync is a simple library that allows you to sync common objects to popular accounting '\
+                     'software like QuickBooks Online, Xero, NetSuite, etc.'
+  spec.homepage = 'https://github.com/LedgerSync/ledger_sync'
```

Comment:
`www.ledgersync.dev`

Reply:
Any change needed for the summary or description ?